src/defense/fail2ban.cr

```diff
@@ -1,6 +1,6 @@
 module Defense
   class Fail2Ban
-    def self.filter(discriminator : String, maxretry : Int32 = 0, findtime : Int32 = 0, bantime : Int32 = 0) : Bool
+    def self.filter(discriminator : String, maxretry : Int32 = 0, findtime : Int32 = 0, bantime : Int32 = 0, &) : Bool
```

Comment:
This is an ameba fix needed

<img width="859" alt="Screenshot 2023-12-01 at 07 55 04" src="https://github.com/defense-cr/defense/assets/6045239/349b6149-a4a9-44f4-831b-bb5ec15dd9d8">